### ISSUE TITLE
NTR: Stool occult blood positive (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7887]
New term request Stool occult blood positive (ang_130429025553): Guaiac, specific gastrointestinal bleeding or hemorrhage, GI bleed  UMLS: 0266813

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000221
- label: Positive stool occult blood test
- parents: [Abnormal stool composition (HP:0031685)](https://hpo.jax.org/app/browse/term/HP:0031685)
- pmid: PMID:30725823
- def: The presence of hidden (occult) blood in the stool.
- comment: This test is commonly used for colorectal cancer screening. Blood in stool may be a sign of hemorrhoids, colorectal cancer, or another condition.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7887
- synonyms: Positive fecal occult blood test(NONE)
